packages/oscal-react-library/src/components/OSCALControlPart.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { OSCALControlPart } from "./OSCALControlPart";
+import {
+  OSCALReplacedProseWithParameterLabel,
+  OSCALReplacedProseWithByComponentParameterValue,
+} from "./OSCALControlProse";
+import {
+  exampleModificationSetParameters,
+  exampleModificationSetParametersDecimal,
+} from "../test-data/ModificationsData";
+import {
+  controlImplTestData,
+  controlImplWithDecSmtTestData,
+  controlProseTestData,
+  controlProseDecimalTestData,
+  exampleParams,
+  exampleDecimalParams,
+} from "../test-data/ControlsData";
+import { sspRestData } from "../test-data/SystemData";
+
+const labelTestData = "label-1";
+const labelTestDataDecimal = "label-1.1";
+
+describe("OSCALControlProse", () => {
```

Comment:
I think this test already exists? 

https://github.com/EasyDynamics/oscal-react-library/blob/64fed6bdc4b7592790479698f02faf98aeba62ad/packages/oscal-react-library/src/components/OSCALControlProse.test.js#L24

Reply:
Yeah, this test already exists, I am modifying this test so that when the part is removed the prose should appear with strike-through line.

Reply:
Hm ok. Maybe we should change the `describe` then?

In general, we have been moving to components testing their own functionality (rather than the parent or other components). So `OSCALControlPart` tests render an `OSCALControlPart` and test only its functionality and `OSCALControlProse` tests render an `OSCALControlProse` and test only its functionality. 

I realize this is a draft, but I want to make sure to be on the same page before we go too far in writing tests. It is confusing when we make a change to a very low level component and tests begin to fail in it's parent. In theory, a component should be usable on it's own.

Reply:
You are right, as I said, I adjusted that unit test to test control part functionalities. I just based my initial implementation on tests that have similar functionalities, given that I was not able to run test locally, I first used a test that should passed, as an initial starting point.